src/main/scala/org/broadinstitute/dsde/workbench/sam/api/StandardSamUserDirectives.scala

```diff
@@ -75,7 +95,7 @@ trait StandardSamUserDirectives extends SamUserDirectives with LazyLogging with
       optionalHeaderValueByName(managedIdentityObjectIdHeader).map(_.map(ManagedIdentityObjectId)))
       .as(OIDCHeaders)
       .map { oidcHeaders =>
-        logger.info(s"Auth Headers: $oidcHeaders")
+        logger.debug(s"Auth Headers: $oidcHeaders")
```

Comment:
This makes Sam no longer log the "Auth Headers" message when deployed. Its replaced by the new log message with json payload.